mp4parse_capi/src/lib.rs

```diff
@@ -631,20 +627,12 @@ pub unsafe extern "C" fn mp4parse_get_track_info(
 
     let track = &context.tracks[track_index];
 
-    if let (Some(track_timescale), Some(context_timescale)) = (track.timescale, context.timescale) {
-        let media_time: CheckedInteger<_> = match track
-            .media_time
-            .map_or(Some(Microseconds(0)), |media_time| {
-                track_time_to_us(media_time, track_timescale)
-            }) {
-            Some(time) => time.0.into(),
+    if let (Some(_), Some(_)) = (track.timescale, context.timescale) {
+        let media_time: CheckedInteger<u64> = match track.media_time {
```

Comment:
Need to carry over the `map_or(Some(0))` behaviour here and below for `empty_duration`.